.travis.yml

```diff
@@ -12,6 +12,7 @@ before_script:
   - pip install -r requirements.txt
   - pip install coveralls
   - psql -c 'create database authors;' -U postgres
+  - python manage.py makemigrations  reporter --noinput
```

Comment:
Duplicate line here

Reply:
this is necessary as the migrations for reporter weren't running on Travis without that line